example/src/testApp.cpp

```diff
@@ -45,7 +45,7 @@ void testApp::setup(){
 	ofEnableAlphaBlending();
 
 	ofPixels pixels;
-	ofBitmapStringGetTextureRef().readToPixels(pixels);
+	ofTexture().readToPixels(pixels);
 	ofSaveImage(pixels,"font.bmp");
```

Comment:
hey thanks! can you just remove this 3 lines, they don't make sense at all, it seems just something i was testing for something else and i quickly added it in this example but never removed it